packages/core/src/generatePolicy.js

```diff
@@ -494,13 +505,13 @@ function aggregateDeps({ packageModules, moduleIdToModuleRecord }) {
  * because resolution was skipped for that module
  *
  * @param {string} requestedName
- * @returns {string}
+ * @returns {string | undefined}
  */
 function guessPackageName(requestedName) {
   const isNotPackageName =
     requestedName.startsWith('/') || requestedName.startsWith('.')
   if (isNotPackageName) {
-    return `<unknown:${requestedName}>`
+    return
```

Comment:
the LHS of the `isNotPackageName` assignment feels like it should be in a function exported from `aa`